interface.h

```diff
@@ -224,6 +224,9 @@ extern netdissect_options *gndo;
 #define snaplen     gndo->ndo_snaplen
 #define snapend     gndo->ndo_snapend
 
+#define printstart gndo->ndo_snaplen
+#define printend gndo->ndo_snaplen
```

Comment:
If possible, please change the code to use ndo-> explicitely, rather than introducing new macros.

Reply:
Especially given that there *IS* no `gndo` variable any more.